.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.1.0.dev3
```

Comment:
We're about to ship 2.1.0 (it was already merged into master) so I think this would be `2.2.0.dev1`? You can probably `bumpversion minor`.